.claude/skills/admin-ui/SKILL.md

```
@@ -71,3 +74,28 @@ agent: general-purpose
 ### HTML
 - 스크립트 순서: `auth.js` → `api.js` → `pagination.js` → `{page}.js`
 - 시맨틱 HTML5 + 접근성 속성 (aria-label, role)
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)